### ISSUE TITLE
[BUG] Note to the installation guide

### ISSUE THREAD

**Issue by ArtyomIvlev** [dmontagu/fastapi-utils#340]
Hi! your instalation [guide](https://fastapi-utils.davidmontague.xyz/#installation)  differs your guide in [github](https://github.com/dmontagu/fastapi-utils?tab=readme-ov-file#installation) 
<img width="1089" alt="image" src="https://github.com/user-attachments/assets/226b09f3-44f2-46ba-876a-0c15d64788b7">
<img width="1025" alt="image" src="https://github.com/user-attachments/assets/8b0a1071-20ea-40d3-afe5-19ba6d97ce15">

**Comment by yuval9313** [comment]
Yeah, I know, it is a permission thing, I'm waiting on @dmontagu to allow me redeploy the guide properly
